core/src/main/java/io/kestra/core/runners/VariableRenderer.java

```diff
@@ -34,9 +35,12 @@
 @Singleton
 public class VariableRenderer {
     private static final Pattern RAW_PATTERN = Pattern.compile("\\{%[-]*\\s*raw\\s*[-]*%\\}(.*?)\\{%[-]*\\s*endraw\\s*[-]*%\\}");
+    public static final int MAX_RENDERING_AMOUNT = 100;
 
+    @Getter
```

Comment:
Is it really needed? This looks like a bad idea (same for the other getter)

Reply:
I forgot to remove it